src/_locales/ja/messages.json

```diff
@@ -260,28 +260,28 @@
     "wallets": "仮想通貨ウォレット",
     "untitled_wallet": "無題の財布",
     "your_message": "あなたのメッセージ...",
-    "plugin_file_service_display_name": "",
-    "plugin_file_service_file_key_copied": "",
-    "plugin_file_service_on_insert": "",
-    "plugin_file_service_uploading_preparing": "",
-    "plugin_file_service_uploading_estimating_time": "",
-    "plugin_file_service_uploading_in_minute_remaining": "",
-    "plugin_file_service_uploading_remaining": "",
-    "plugin_file_service_uploading_on_cancal": "",
-    "plugin_file_service_show_more": "",
-    "plugin_file_service_file_key": "",
-    "plugin_file_service_unencrypted": "",
-    "plugin_file_service_signing_failed": "",
-    "plugin_file_service_on_encrypt_it": "",
-    "plugin_file_service_on_show_more": "",
-    "plugin_file_service_legal_text": "",
-    "plugin_file_service_legal_terms_link": "",
-    "plugin_file_service_legal_policy_link": "",
-    "plugin_file_service_drop_indicator": "",
-    "plugin_file_service_drop_hint_1": "",
-    "plugin_file_service_drop_hint_2": "",
-    "plugin_file_service_drop_here": "",
-    "plugin_file_service_error_101": "",
-    "plugin_file_service_error_102": "",
-    "plugin_file_service_on_change_file": ""
+    "plugin_file_service_display_name": "プラグイン: ファイル転送",
+    "plugin_file_service_file_key_copied": "ファイルの鍵がコピーされました",
+    "plugin_file_service_on_insert": "ファイルを使用する",
+    "plugin_file_service_uploading_preparing": "準備しています",
+    "plugin_file_service_uploading_estimating_time": "予定準備時間...",
```

Comment:
I interpret "予定準備時間" to mean "Estimated preparation/preparing time" and it could hardly convey that the software is working on producing an estimation. Could we use terms like "予定時間計算中" or "予定転送時間計算中"?

Reply:
Oops. This is my mistake. "転送時間計算中" or "予定転送時間計算中" means collectly.